javascript/javascript1/week1/housey_pricey.js

```diff
@@ -0,0 +1,25 @@
+let width = parseInt(prompt('Enter width of the property'));
+let deep = parseInt(prompt('Enter depth of the property'));
+let high = parseInt(prompt('Enter height of the property'));
+let gardenSizeInM2 = parseInt(prompt('Enter size of garden of the property'));
+let price = parseInt(prompt('Enter your house price'));
+let volumeInMeters = width * deep * high;
+let housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
```

Comment:
since this will be the same for both cases mentioned in the HW, you can place these two inside your function so you won't need to repeat it for different scenarios and numbers.